app/scripts/views/password_strength/password_with_strength_balloon.js

```diff
@@ -67,11 +56,31 @@ const PasswordWithStrengthBalloonView = FormView.extend({
     const { hasEnteredPassword, isValid } = this.model.toJSON();
 
     if (hasEnteredPassword && ! isValid) {
-      this.$el.addClass('invalid');
+      this.$el.addClass('invalid').attr('aria-invalid', 'true');
```

Comment:
Is this pattern of setting/unsetting `aria-invalid` at the same time as the `invalid` class something we should be applying elsewhere too? If so is it worth pulling out somewhere more generally usable?

Reply:
That's a good idea @philbooth, we *will* need it for https://github.com/mozilla/fxa-content-server/issues/6310. Extracting now.